src/app/modules/shared_models/section_model.nim

```diff
@@ -204,7 +204,7 @@ QtObject:
     of ModelRole.AmIBanned:
       result = newQVariant(item.amIBanned)
 
-  proc isItemExist(self: SectionModel, id: string): bool =
+  proc isItemExist*(self: SectionModel, id: string): bool =
```

Comment:
I know you didn't write this function, but that name is not good English. Maybe it can be renamed to just `itemExists` or `doesItemExist`.